### PR TITLE
update checks and clean up naming standards

### PR DESCRIPTION
**Pull request by cdaniluk**

### CODE REVIEW

**Review by sblack4 — APPROVED** [review]
The only thing I don't like is the deletion of my beloved `.terraform-version` file

File: .terraform-version [hunk 1]
```diff
@@ -1 +0,0 @@
-0.12.13
```
Comment (smiller171) [inline .terraform-version:1]: I thought we were keeping these but reformatting as `latest:^0.13`?

File: main.tf [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
 }
 
+data "aws_region" "current" {
+}
+
 locals {
-  account_id = data.aws_caller_identity.current.account_id
+  account_id  = data.aws_caller_identity.current.account_id
+  bucket_name = var.bucket_name == null ? "${local.account_id}-${local.region}-${var.bucket_suffix}" : var.bucket_name
+  region      = data.aws_region.current.name
```
Comment (smiller171) [inline main.tf:10]: I thought we prefered this syntax?

```suggestion
  account_id  = data.aws_caller_identity.current.account_id
  region      = data.aws_region.current.name
  
  bucket_name = coalesce(
    var.bucket_name,
    "${local.account_id}-${local.region}-${var.bucket_suffix}"
  )
```

File: variables.tf [hunk 3]
```diff
@@ -1,29 +1,17 @@
-variable "bucket_prefix" {
-  default     = "awsconfig"
-  description = "Name of the S3 bucket to log to"
+########################################
+# General Vars
+########################################
+
+variable "bucket_name" {
+  default     = null
+  description = "Name of the S3 bucket to record to (do not use with multi-region module)"
```
Comment (smiller171) [inline variables.tf:7]: Is this still relevant? I thought you didn't create a multi-region module after all?

**Review by smiller171 — COMMENTED** [review]
Had a few questions in here.

**Review by jjduverge — APPROVED** [review]

**Pull request merged**